design/specs/column-extraction.md

```diff
@@ -8,87 +8,62 @@ editor: markdown
 dateCreated: 2022-07-22T11:12:59.313Z
 ---
 
-# Context
-Tables must be split into smaller tables during the normalization process to ensure data integrity. To make this process easier, the product should include capabilities that allow users to restructure data by splitting, merging, and combining existing tables.
-Initially, just table splitting will be implemented, so that a user can select columns from an existing table and create a new one.
+## Context
 
-# Splitting a Table by Extracting Columns
-To create a new table from an existing one, select the columns to include in the new table. These columns must not be primary keys for the parent table (although they can be foreign keys to another table). Once selected, the inspector panel should display a list of possible 'Selection Actions,' with the 'New Table from Columns' option allowing the user to extract the columns into a new table.
+In data modeling, it's usual practice to split data up into related tables. In the context of Mathesar, one way of doing this is by extracting columns from a table into a new table. This process is known as column extraction.
```

Comment:
Nitpick, but it might be useful to explain why data is split up into related tables (one table per type of object, data deduplication, etc.)